tracsubtickets/checker.py

```diff
@@ -89,7 +88,8 @@ def main(args=sys.argv[1:]):
 
     # check all the environments
     for env in envs:
-        check_subtickets(env)
+        with env.db_transaction as db:
```

Comment:
Removing `db` parameter from `check_subtickets` would make this context manager unnecessary.